### ISSUE TITLE
error: "install packages 'R.utils' required"

### ISSUE THREAD

**Issue by gledauio** [comorment/magma#1]
![image](https://user-images.githubusercontent.com/98422962/222165673-75bf4cb7-7f5e-4a9d-95f7-d05327202eab.png)

**Comment by espenhgn** [comment]
Hi @gledauio. Can you post a list of libraries you require from your script(s) if we should rebuild this container? Looks like this one contains a fairly minimalist R environment.

**Comment by gledauio** [comment]
Hi Espen, the problem is that when you call unziped files LAVA is running ok when you call gz files then I got this error. Maybe it requires this package that is related to gz files. So I don't think it will require more other libraries because they already are called since the script is working ok while avoiding calling gz files.

**Comment by gledauio** [comment]
A list of required libraries for LAVA is represented here: https://github.com/josefin-werme/LAVA

**Comment by espenhgn** [comment]
Yes, I could see from the above that `R.utils` is missing as it is merely a suggested dependency for `data.table`. 
I'll make a PR with a rebuilt container that includes `R.utils`. As far as I can tell, everything else is included. The quickfix I'm sure you figured out; just unzip the sumstats file ;-)

**Comment by espenhgn** [comment]
Hi @gledauio. I've opened #2 with a fix. Can you test it? The corresponding .sif file is this one: https://github.com/comorment/magma/blob/fix-1/containers/magma.sif

**Comment by gledauio** [comment]
> Hi @gledauio. I've opened #2 with a fix. Can you test it? The corresponding .sif file is this one: https://github.com/comorment/magma/blob/fix-1/containers/magma.sif

Thanks Espen.
